components/movementsensor/gpsrtkserial/gpsrtkserial.go

```diff
@@ -583,9 +557,6 @@ func (g *rtkSerial) readSatsInView(ctx context.Context) (int, error) {
 
 // Properties passthrough.
 func (g *rtkSerial) Properties(ctx context.Context, extra map[string]interface{}) (*movementsensor.Properties, error) {
-	g.mu.Lock()
-	defer g.mu.Unlock()
```

Comment:
Note to myself: these changes are the important part: even if we locked the mutex for the entirety of the for loop above, by not locking it here we don't care about how slow the call to `g.ntripClient.Client.GetStream()` is.